src/files/dto/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './get-file-url-response.dto';
+export * from './upload-file-response.dto';
+export * from './delete-file-response.dto';
```

Comment:
In `.vscode/settings.json` you can add this for automatic import organization on save:

```
  "editor.codeActionsOnSave": {
    "source.organizeImports": "explicit"
  },
```